stdlib/obj.ml

```diff
@@ -30,6 +30,9 @@ external size : t -> int = "%obj_size"
 external reachable_words : t -> int = "caml_obj_reachable_words"
 external field : t -> int -> t = "%obj_field"
 external set_field : t -> int -> t -> unit = "%obj_set_field"
+external compare_and_swap_field : t -> int -> t -> t -> bool
+  = "caml_obj_compare_and_swap"
+external is_shared : t -> bool = "caml_obj_is_shared"
```

Comment:
Consider for removal.

Reply:
Tracking this for post-merge fixing in https://github.com/ocaml/ocaml/issues/10861